### PR TITLE
glossary: Remove "leaf" and "hierarchy" from container namespace definition

### PR DESCRIPTION
**Pull request by wking**
Namespaces are not all hierarchical and processes aren't always in leaves.

Spun off from #795.

### CODE REVIEW

**Comment by wking** [comment]
Pulled in #853 based on [today's meeting][1].

[1]: http://ircbot.wl.linuxfoundation.org/meetings/opencontainers/2017/opencontainers.2017-05-31-21.02.log.html#l-88

File: glossary.md [hunk 1]
```diff
@@ -30,9 +30,8 @@ It reads the [configuration files](#configuration) from a [bundle](#bundle), use
 
 ## <a name="glossaryRuntimeNamespace" />Runtime namespace
 
-On Linux, a leaf in the [namespace][namespaces.7] hierarchy from which the [runtime](#runtime) process is executed.
-New container namespaces will be created as children of the runtime namespaces.
-
+On Linux, the namespaces from which new [container namespaces](#container-namespace) are [created](config-linux.md#namespaces) and from which some configured resources are accessed.
+Examples of resources retrieved from this namespace include [`linux.namespaces[].path`](config-linux.md#namespaces) and the [`resctrl` psuedo-filesystem used for `linux.intelRdt`](config-linux.md#intelrdt).
```
Comment (crosbymichael) [inline glossary.md]: Please remove this line.

Comment (mrunalp) [inline glossary.md]: Remove this line.

**Comment by mrunalp** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**